src/pages/microblocks/[id].vue

```diff
@@ -18,14 +20,17 @@ import { useRoute } from 'nuxt/app'
 import { useMicroblockDetailsStore } from '@/stores/microblockDetails'
 import PageHeader from '@/components/PageHeader'
 import MicroblockDetailsPanel from '@/components/MicroblockDetailsPanel'
+import MicroblockTransactionsPanel from '~/components/MicroblockTransactionsPanel'
 
 const microblockDetailsStore = useMicroblockDetailsStore()
 const { microblockDetails } = storeToRefs(microblockDetailsStore)
 const { fetchMicroblock } = microblockDetailsStore
 const route = useRoute()
 
-await useAsyncData(async() => {
-  await fetchMicroblock(route.params.id)
-  return true
-})
+if (process.client) {
```

Comment:
Without this, the hydration problem will occur again

Reply:
I think only the transactions need to be fetched on the client side because only the transactions table size depends on the resolution-based limit. The following change should keep the issue fixed while allowing to fetch the microblock details on the server-side:
```
diff --git a/src/components/MicroblockTransactionsPanel.vue b/src/components/MicroblockTransactionsPanel.vue
index e57f0bf..380a7fa 100644
--- a/src/components/MicroblockTransactionsPanel.vue
+++ b/src/components/MicroblockTransactionsPanel.vue
@@ -42,7 +42,7 @@ function loadNextTransactions() {
 }
 
 if (process.client) {
-  await fetchMicroblockTransactions({
+  fetchMicroblockTransactions({
     limit: limit.value,
     microblockHash: route.params.id,
   })
diff --git a/src/pages/microblocks/[id].vue b/src/pages/microblocks/[id].vue
index b728ee4..c69073e 100644
--- a/src/pages/microblocks/[id].vue
+++ b/src/pages/microblocks/[id].vue
@@ -27,10 +27,8 @@ const { microblockDetails } = storeToRefs(microblockDetailsStore)
 const { fetchMicroblock } = microblockDetailsStore
 const route = useRoute()
 
-if (process.client) {
-  await useAsyncData(async() => {
-    await fetchMicroblock(route.params.id)
-    return true
-  })
-}
+await useAsyncData(async() => {
+  await fetchMicroblock(route.params.id)
+  return true
+})
 </script>
 ```

Reply:
Thanks for the hint, I made it work. Sorry, I didn't pay too much attention to the previous solution.
Also applied this sanitization for the Keyblock Microblocks table